### ISSUE TITLE
Blank page when running in production environment

### ISSUE THREAD

**Issue by dlford** [standardnotes/web#252]
Hello,

Everything works fine when running in development environment, but I get a blank page when running in production, with a failure to load `/javascripts/compiled.min.js`, the issue is resolved by changing the config for the production environment as such: `config.assets.compile = true`.  It is entirely possible I'm missing a step here but I'm not seeing it.

Works: `rails s -b 0.0.0.0 -p 3000`

Doesn't work: `rails s -b 0.0.0.0 -p 3000 -e production`

Thank you for your time and efforts.

dlford

**Comment by mobitar** [comment]
You may need to run `rails assets:precompile`..

**Comment by dlford** [comment]
Thanks for your reply, I tried running `rails assets:precompile`, the results were the same.  But I also notice that extensions weren't working either, and viewing the package.json for an extension in browser came back not found.  That lead me to read more into the production configuration and I discovered that adding `RAILS_SERVE_STATIC_FILES=true` the the `.env` file fixed both issues, and I was able to change `config.assets.compile` back to `false` with no issue.  Which I suppose makes perfect sense because the `compiled.min.js` should be considered a static file, it is my opinion that maybe the production configuration should serve static files by default?  Anyway I will close this issue since it is resolved for me.

Thanks again!


dlford
